phpcs.xml

```diff
@@ -1,24 +1,42 @@
 <?xml version="1.0"?>
 <ruleset name="Yoast SEO">
-	<description>Yoast SEO rules for PHP_CodeSniffer</description>
+    <description>Yoast SEO rules for PHP_CodeSniffer</description>
 
     <file>.</file>
 
-	<exclude-pattern>tests/*</exclude-pattern>
-	<exclude-pattern>vendor/*</exclude-pattern>
-	<exclude-pattern>node_modules/*</exclude-pattern>
-	<exclude-pattern>deprecated/*</exclude-pattern>
-	<exclude-pattern>languages/*</exclude-pattern>
+    <exclude-pattern>tests/*</exclude-pattern>
+    <exclude-pattern>vendor/*</exclude-pattern>
+    <exclude-pattern>node_modules/*</exclude-pattern>
+    <exclude-pattern>deprecated/*</exclude-pattern>
+    <exclude-pattern>languages/*</exclude-pattern>
 
-    <arg name="extensions" value="php" />
+    <arg name="extensions" value="php"/>
     <arg value="sp"/>
 
     <rule ref="Yoast">
-        <exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification" /><!-- TODO audit and fix nonces -->
-        <exclude name="WordPress.WP.PreparedSQL.NotPrepared" /><!-- TODO audit raw queries -->
+        <exclude name="WordPress.WP.PreparedSQL.NotPrepared"/><!-- TODO audit raw queries -->
+        <exclude name="WordPress.Functions.DontExtract" /><!-- TODO audit usage & replace -->
```

Comment:
As the `extract` rule is excluded, why are the lines related to it still being wrapped in `ignore` annotations ?